src/main/java/com/grimbo/chipped/registry/ChippedBlocks.java

```diff
@@ -321,7 +321,7 @@ public static void register() {
     }
 
     //Registries
-    private static void registerSpecialLantern(List<Block> list, String name, VoxelShape hangingShape, VoxelShape normalShape, int... indices) {
+    private static void registerSpecialLantern(List<Block> list, String name, VoxelShape hangingShape, VoxelShape normalShape, int @NotNull ... indices) {
```

Comment:
@NotNull is unnecessary on a varargs argument.

Reply:
fax
im just gonna remove all of the @NotNull
i dont even remember why i added them

Reply:
No, NotNulls are good, but in this context it's just unnecessary